services/system/AuthSig/CMakeLists.txt

```diff
@@ -5,6 +5,9 @@ function(add suffix)
     add_library(Spki${suffix} src/Spki.cpp)
     target_link_libraries(Spki${suffix} PUBLIC -lbotan-3 services_system${suffix} psibase${suffix} exception-stub${suffix})
     target_include_directories(Spki${suffix} PUBLIC ${ROOT_BINARY_DIR}/wasm/deps/include/botan-3 include)
+    target_link_options(Spki${suffix} PUBLIC 
+        -Wl,-z,stack-size=32768
```

Comment:
@swatanabe 

I am guessing that this change is responsible for the CI/CD failure: `std::exception: obj too big`
https://github.com/gofractally/psibase/blob/98f3198a66f4754b29468ce783bd7a3496fd6219/libraries/triedent/include/triedent/cache_allocator.hpp#L139

It happens during the execution of `chain.finish_block();` in the test_contract_2 unit test:
https://github.com/gofractally/psibase/blob/98f3198a66f4754b29468ce783bd7a3496fd6219/rust/test_contract_2/src/lib.rs#L111

But I'm not sure how to address it

Reply:
Future reader: the problem was that the rust tester doesn't automatically make new blocks, so I added a call to `start_block` after boot, which ensures that the boot block is separated from the transactions added to the block during a test.
Might need to revisit the tester if the boot block itself becomes too big.